lab_3_ann_retriever/main.py

```diff
@@ -253,6 +388,18 @@ def index_documents(self, documents: list[str]) -> bool:
 
         In case of corrupt input arguments, False is returned.
         """
+        if not isinstance(documents, list):
+            return False
+
+        self._documents = documents
+        temp_vector_docs = []
+        for doc in self._documents:
+            vector_doc = self._index_document(doc)
+            if not vector_doc is Vector:
```

Comment:
can you use `isinstance()`?

Reply:
![image](https://github.com/user-attachments/assets/4b53c6b4-05ca-429d-9c01-1f17d0a82575)
No, this causes an error